exporter/stats/stackdriver/stackdriver.go

```diff
@@ -397,3 +399,17 @@ func equalAggWindowTagKeys(md *metricpb.MetricDescriptor, agg stats.Aggregation,
 
 	return nil
 }
+
+func sanitize(s string) string {
+	if len(s) > labelKeySizeLimit {
+		s = s[:labelKeySizeLimit]
+	}
+	return strings.Map(sanitizeRune, s)
+}
+
+func sanitizeRune(r rune) rune {
+	if unicode.IsLetter(r) || unicode.IsDigit(r) || r == 95 {
```

Comment:
Could we please use '_' instead of the code for an underscore?

Reply:
Sure, done.